packages/installer/cos.sh

```diff
@@ -539,47 +545,47 @@ validate_device()
 ## UPGRADER
 
 find_partitions() {
-    STATE=$(blkid -L COS_STATE || true)
-    if [ -z "$STATE" ]; then
+    _STATE=$(blkid -L COS_STATE || true)
+    if [ -z "$_STATE" ]; then
         echo "State partition cannot be found"
         exit 1
     fi
 
-    OEM=$(blkid -L COS_OEM || true)
+    _OEM=$(blkid -L COS_OEM || true)
 
-    PERSISTENT=$(blkid -L COS_PERSISTENT || true)
-    if [ -z "$PERSISTENT" ]; then
+    _PERSISTENT=$(blkid -L COS_PERSISTENT || true)
+    if [ -z "$_PERSISTENT" ]; then
         echo "Persistent partition cannot be found"
         exit 1
     fi
 
-    COS_ACTIVE=$(blkid -L COS_ACTIVE || true)
-    if [ -n "$COS_ACTIVE" ]; then
-        CURRENT=active.img
+    _COS_ACTIVE=$(blkid -L COS_ACTIVE || true)
+    if [ -n "$_COS_ACTIVE" ]; then
+        _CURRENT=active.img
     fi
 
-    COS_PASSIVE=$(blkid -L COS_PASSIVE || true)
-    if [ -n "$COS_PASSIVE" ]; then
-        CURRENT=passive.img
+    _COS_PASSIVE=$(blkid -L COS_PASSIVE || true)
+    if [ -n "$_COS_PASSIVE" ]; then
+        _CURRENT=passive.img
     fi
 
-    if [ -z "$CURRENT" ]; then
+    if [ -z "$_CURRENT" ]; then
         # We booted from an ISO or some else medium. We assume we want to fixup the current label
         read -p "Could not determine current partition. Do you want to overwrite your current active partition? (CURRENT=active.img) [y/N] : " -n 1 -r
```

Comment:
See here. `CURRENT` can actually be used to avoid prompting and select the target image. Maybe this one needs a prefix too.